conf/README.md

```diff
@@ -57,6 +57,14 @@ run it belongs here.
 * `log_utilization` - Enable logging of cluster utilization metrics every 10 seconds. Set via --log-cluster-utilization
 * `use_ocs_worker_for_scale` - Use OCS workers for scale testing (Default: false)
 * `load_status` - Current status of IO load
+* `skip_reason_test_found` - In the case of cluster left unhealthy, used to determine the
+test case that is likely to cause that
+* `skipped_tests_ceph_health` - The number of tests that got skipped due to Ceph being unhealthy
+* `number_of_tests` - The number of tests being collected for the test execution
+* `skipped_on_ceph_health_ratio` - The ratio of tests skipped due to Ceph unhealthy against the
+number of tests being collected for the test execution
+* `skipped_on_ceph_health_threshold` - The allowed threshold for the ratio of tests skipped due to Ceph unhealthy against the
+number of tests being collected for the test execution
```

Comment:
Indent, also I would mention here the default value and maybe that in case of acceptance suite, this will be always set to 0.